src/components/Card.js

```diff
@@ -1,62 +1,90 @@
 class Card {
-  constructor({ name, link }, cardSelector, handleCardImage) {
+  constructor( {
+    name, 
+    link,  
+    isLiked,
+    _id,
+  },
+    cardSelector,
+    handleCardImage,
+    handleDelete,
+    userId,
+    handleCardLike,
+
+  ) {
     this._name = name;
     this._link = link;
+    this._isLiked = isLiked;
+    this._cardId = _id;
     this._cardSelector = cardSelector;
     this._handleCardImage = handleCardImage;
+    this._handleDeleteCard = handleDelete;
+    this._userId = userId;
+    this._handleCardLike = handleCardLike
+
+  }
+
+  getView() {
+    this._cardElement = this._getTemplate();
+    this._handleCardDeleteButton = this._cardElement.querySelector(".card__delete-button");
+    this._cardLikeButton = this._cardElement.querySelector(".card__like-button");
+    this._setEventListeners();
+    this.renderLikes();
+    this._cardElement.querySelector(".card__like-button");

```

Comment:
`    this._cardElement.querySelector(".card__like-button");
`

Please, delete line 33. It has no meaning